README.md

```diff
@@ -1 +1,85 @@
-# learning-app
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Learn It] <a name="about-project"></a>
+
+**[Learn It]** is a an online educative website that provides slides, videos, questions and answers to users. We also host a weekly class with one of our best Lecturers.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS, Javascript and linter files
+
+### Key Features <a name="key-features"></a>
+
+- **[linter files(debugging tools)]**
+- **[ html and css template]**
+- **[ Javasript]**
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Install git on your computer
+
+-code editor
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+-git@github.com:Mylo16/Learning-app.git
+
+### Live Demo
+
+-https://mylo16.github.io/
+
+
+## 👥 Author <a name="authors"></a>
+
+
+- GitHub: [@Mylo16](https://github.com/Mylo16)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- In the near future I will be adding the other sections like questions, videos, slides etc
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project show your support by giving a ⭐️
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Cindy Shin. The idea behind this project design was extracted from her design on 2015 CC global summit at beehance
+Secondly, would like to thank the entire microverse staff and students for such a wonderful coding camp
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- Hey, kindly add the license file as the link is currently broken to improve your readme file.